src/components/list/list.ios.scss

```diff
@@ -182,6 +182,10 @@ $list-ios-header-background-color:    transparent !default;
   background: $list-ios-header-background-color;
 }
 
+[dir="rtl"] .list-header-ios {
+  padding-right: $list-ios-header-padding-left;
```

Comment:
@manucorporat 
@sijav and I had this discussion before.
Changing it to `start` will require support for `left`. which is possible and awesome, but should imo be a different PR, changing all relevant variables to `start`/`end` and adding the correct depredations, so in v4 can have a clean variables naming